osmaxx-py/config/settings/common.py

```diff
@@ -381,8 +381,6 @@
     # to request access to osmaxx (access_denied page)
     'account_manager_username': env.str('ACCOUNT_MANAGER_USERNAME', default='admin'),
     'CONVERSION_SERVICE_URL': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_URL', default='http://localhost:8901/api/'),
-    'CONVERSION_SERVICE_USERNAME': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_USERNAME'),
-    'CONVERSION_SERVICE_PASSWORD': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_PASSWORD'),
```

Comment:
Acceptable, yes, because even if this password would happen stay unreplaced on production, it would just cause the UI to try to use it to access the service. It would not cause the service or UI to be accessible using this default password.

But still a bit suboptimal from a debugging perspective.